docs/CHANGELOG.rst

```diff
@@ -14,6 +14,9 @@ Changed
 * Decorate all tests that currently fail on Docker with ``skipDockerFailure``
 * Require Resolwe's ``master`` git branch
 * Put packaging tests in a separate Tox testing environment
+* Require Resolwe 1.2.0
```

Comment:
Please, remove this line. This is a mistake of the otherwise very useful git's 'union' merge driver.